### ISSUE TITLE
Forside av/på-innstilling?

### ISSUE THREAD

**Issue by sda030** [NIFU-NO/nifutypst#28]
Forside og kolofon er fint når man har en mer formell rapport. Men for institusjonsspesifikke rapporter kan det bli rart (og tungvint for de oppdragsgivere som ønsker å bla gjennom flere av disse for å sammenligne). include_front_end_matter: false hadde vært nyttig? En kortere variant som likevel beholder alt av formattering.

Dette er kanskje såpass enkelt at jeg kan prøve på det i påsken.

**Comment by henrikkarlstrom** [comment]
God ide! Bør ikke være vanskeligere enn å lage et flagg som du sier og så putte en if-statement rundt front matter i template-fila. 

...men det får meg til å tenke at kanskje front matter bør trekkes ut i en egen fil/funksjon som kan kalles fra typst-template.typ, så den ikke ender med å bli en gigantisk fil med masse komplisert control flow...

**Comment by sda030** [comment]
Ja, og kan jo til og med tenkes at man kun ønsker front matter uten innhold, dersom resten er blitt ordnet fra før. Selv om det er sjeldent da.

**Comment by sda030** [comment]
> Ja, og kan jo til og med tenkes at man kun ønsker front matter uten innhold, dersom resten er blitt ordnet fra før. Selv om det er sjeldent da.

På den andre siden hadde det vært veldig kjekt om man bare trengte å sette `report_type: ""` for å ikke få rapport eller arbeidsnotat-forsidene.
